statsmodels/nonparametric/tests/test_kernels.py

```diff
@@ -101,22 +101,26 @@ def test_smoothconf(self):
 
         #assert_allclose(fitted, res_fitted, rtol=0, atol=1e-6)
 
-    def t_est_smoothconf_data(self):
+    @pytest.mark.smoke  # TOOD: make this an actual test?
+    def test_smoothconf_data(self):
```

Comment:
this one might be expensive
maybe better not to run it

Reply:
could also add slow marker